src/Symfony/Component/HttpFoundation/Session/Storage/NativeSessionStorage.php

```diff
@@ -203,7 +203,15 @@ public function regenerate($destroy = false, $lifetime = null)
             $this->metadataBag->stampNew();
         }
 
-        return session_regenerate_id($destroy);
+        $isRegenerated = session_regenerate_id($destroy);
+
+        if (PHP_VERSION_ID >= 70000) {
```

Comment:
why not simply do it for every php version if the old behavior was never intended to work this way?